conda/recipes/cusignal/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - numpy>=1.17.3
     - boost
     - numba>=0.49.0
-    - cupy>=7.2.0,<8.0.0a0
+    - cupy>=7.2.0,<9.0
```

Comment:
We got the tip to do `<9.0.0a0` so we don't pick up alpha versions of 9.x accidentally, though I'm not 100% sure that's necessary.